lib/imageResize.ts

```diff
@@ -207,7 +220,7 @@ export const ImageResize = Image.extend({
           const $dot = document.createElement('div');
           $dot.setAttribute(
             'style',
-            `position: absolute; width: 9px; height: 9px; border: 1.5px solid #6C6C6C; border-radius: 50%; ${dotsPosition[index]}`
+            `position: absolute; width: ${isMobile ? 16 : 10}px; height: ${isMobile ? 16 : 10}px; border: 1.5px solid #6C6C6C; border-radius: 50%; ${dotsPosition[index]}`,
```

Comment:
I think it would be best to continue using 9px on the web env.

Reply:
😄

Reply:
Hi, @22JH Thanks for the update! Just to clarify, I just wanted to replace 10px with 9px for the web version.

Could you also adjust the mobile width and height to 16px again? And please double-check dotsPosition as well.

ex)
```js
 `position: absolute; width: ${isMobile ? 16 : 9}px; height: ${isMobile ? 16 : 9}px; border: 1.5px solid #6C6C6C; border-radius: 50%; ${dotsPosition[index]}`,
```

Reply:
I must have misread this lol
I've adjusted it to 10px for web (min 768px) and 16px for mobile.